manifests/certonly.pp

```diff
@@ -58,6 +59,7 @@
   Array[String[1]]                          $environment          = [],
   Boolean                                   $manage_cron          = false,
   Boolean                                   $suppress_cron_output = false,
+  Boolean                                   $log_cron_output      = false,
```

Comment:
These two are mutually exclusive.  Maybe add a test condition that raise an error if both or true?  Or maybe replace them with a single setting `Enum['suppress', 'log', 'ignore'] $cron_output = 'ignore'` ?